src/live_view/index.html

```diff
@@ -81,33 +216,43 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.16.0/prism.js"></script>
 
 <header id="header">
-    <button id="enableButton">Enable Live View</button>
-    <button id="disableButton" style="display: none">Disable Live View</button>
+    <h1>APIFY <span>Live view</span></h1>
```

Comment:
APIFY => Apify

Reply:
It's in the typical APIFY font that's in our logo, so I'll keep it for now, it looks good IMO. @vbartonicek will do one more round on the UI, so we can tweak it later.